config_defaults.yml

```diff
@@ -41,6 +41,8 @@ appveyor.yml:
       RUBY_VER: 21-x64
   test_script:
   - "bundle exec rake spec SPEC_OPTS='--format documentation'"
+MAINTAINERS.md:
+  maintainers: "Puppet Forge Modules Team <forge-modules@puppet.com>"
```

Comment:
Yes.  Override the email in the module.  This is more in the spirit of the original request as each repo should have a maintainer specified.